style.css

```diff
@@ -9,59 +9,61 @@
   src: url(./elements/fonts/Poppins-Light.ttf);
 }
 
+* {
+  box-sizing: border-box;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+}
+
 header {
   display: flex;
   justify-content: space-between;
-  width: 375px;
-  left: 0;
-  right: 0;
-  top: 44px;
-  bottom: 0;
+  width: 330px;
+  margin: auto;
+}
+
+.logo a:link {
+  font-weight: 700;
+  color: #6070ff;
+  text-decoration: none;
 }
 
-.parent1 {
+.intro {
   display: flex;
   flex-direction: column;
   align-items: flex-start;
-  width: 376px;
-  height: 100vh;
+  padding: 16px;
+  background: #fff;
+  border-radius: 16px;
+  max-width: 327px;
+  margin: auto;
+  margin-bottom: 70px;
   background-image: url(./elements/bglmnt.png);
   background-repeat: no-repeat;
 }
```

Comment:
Thanks to padding and margins your background seems a little off, you can't bypass margins, but you can ignore padding in your background-image by adding the property and value: `box-sizing: content-box;`, kindly add that line to this class to make it look similar to the Figma design 💯